src/main/java/com/minecolonies/coremod/colony/CitizenData.java

```diff
@@ -762,6 +771,12 @@ public void serializeViewNetworkData(@NotNull final ByteBuf buf)
         {
             buf.writeInt(0);
         }
+
+        buf.writeInt(colony.getID());
+
+        NBTTagCompound compound = new NBTTagCompound();
+        compound.setTag("inventory", inventory.writeToNBT(new NBTTagList()));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use already-defined constant 'TAG_INVENTORY' instead of duplicating its value here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1192)